magefile.go

```diff
@@ -1147,6 +1159,10 @@ func flattenDependencies(requiredPackages []string, packageVersion, archivePath,
 		}
 		matches = append(matches, zipMatches...)
 
+		// never flatten any python wheels, the packages.yml and docker should handle
+		// those specifically so that the python wheels are installed into the container
+		matches = removePythonWheels(matches, packageVersion)
```

Comment:
Is there an alternative to adding the PythonWheel binary specs to the `ExpectedBinaries` just to remove them from the flattened directory? The connectors are added to `ExpectedBinaries` just to download the archive ?
If the handling of connectors is different from the other components' binaries we package with agent maybe we can separate the handling with a `collectAdditionalDependencies` target  after flattening or something similar ? 
My understanding is also that these dependencies are not needed to create an agent archive but get installed when we run `docker build`, maybe we can move the processing at that stage ?